[DOC release] document how to submit bugfixes for past releases

EricSchank opened the pull request:
Added note on how to tag and PR against older releases.

EricSchank committed:
[DOC release] document bugfixes for past releases

Added note on how to tag and PR against older releases.

rwjblue commented:
I am not 100% sure how to tweak this, but we need to ensure that it is clear that this is:

* Somewhat unusual (aka not something we do very much of).
* For current stable releases or beta releases you target master.

EricSchank committed:
[DOC release] document rare past release bugfixes

Indicate that almost all `BUGFIX`es should be against `master` with `beta` tag.

EricSchank commented:
Incorporated feedback - now indicates that most BUGFIXes should be against master.

rwjblue commented:
Awesome, thank you!
